bookwyrm/templates/settings/schedules.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'settings/layout.html' %}
+{% load i18n %}
+{% load humanize %}
+{% load utilities %}
+
+{% block title %}
+{% trans "Scheduled tasks" %}
+{% endblock %}
+
+{% block header %}
+{% trans "Scheduled tasks" %}
+{% endblock %}
+
+{% block panel %}
+
+<div class="block content">
+    <h3>{% trans "Tasks" %}</h3>
+    <div class="table-container">
+        <table class="table is-striped is-fullwidth">
+            <tr>
+                <th>
+                    {% trans "Name" %}
+                </th>
+                <th>
+                    {% trans "Celery task" %}
+                </th>
+                <th>
+                    {% trans "Date changed" %}
+                </th>
+                <th>
+                    {% trans "Last run at" %}
+                </th>
+                <th>
+                    {% trans "Schedule" %}
+                </th>
+                <th>
+                    {% trans "Schedule ID" %}
+                </th>
+                <th>
+                    {% trans "Enabled" %}
+                </th>
+            </tr>
+            {% for task in tasks %}
+            <tr>
+                <td>
+                    {{ task.name }}
+                </td>
+                <td class="overflow-wrap-anywhere">
+                    {{ task.task }}
+                </td>
+                <td>
+                    {{ task.date_changed }}
+                </td>
+                <td>
+                    {{ task.last_run_at }}
+                </td>
+                <td>
+                    {% firstof task.interval task.crontab "None" %}
+                </td>
+                <td>
+                    {{ task.interval.id }}
+                </td>
+                <td>
+                    {{ task.enabled|yesno }}
```

Comment:
Currently there does not seem to be a way to _disable_ `celery.backend_cleanup` or `check-for-updates`.

Now, one might argue that they shouldn't be disabled, but it seems a bit odd that you have to manually turn on `check-for-updates` but then you can't turn it off.

Reply:
yeah, I agree, I think it should be a subsequent PR because celery.backend_cleanup might be the kind of the thing that it's Not Good to turn off

Reply:
actually but now that I saw that, if someone has it running every minute, they're probably going to be displeased not to be able to stop it.

Reply:
I guess we could trigger a big scary banner if it's ever turned off. I don't think this needs to be a blocker for this PR, just wanted to note it for future improvement.